auto-value-gson/src/main/java/com/ryanharter/auto/value/gson/AutoValueGsonExtension.java

```diff
@@ -349,7 +350,7 @@ public String generateClass(Context context, String className, String classToExt
       if (!seenTypes.contains(property.type)) {
         fields.put(property.type,
                 FieldSpec.builder(adp,
-                    nameAllocator.newName(simpleName(property.type)) + "_adapter", PRIVATE, FINAL)
+                                  nameAllocator.newName(simpleName(property.type)) + "_adapter", PRIVATE, VOLATILE)
```

Comment:
nit: The current convention in this project is for parameter continuation to have two tabs (8 spaces), not align with other parameters.

Reply:
My apologies, I didn't notice. Will fix!

Reply:
You know what, I was wrong, the project uses 2 space indentation, so it's actually 4 spaces for the continuation.  To match the `.build()` line.  Sorry for the mixup.

Reply:
I just pushed another change. Hopefully I got this one right - I put 4 spaces for both the parameter continuation indentation and for the following line starting with ".".